dev/intro-two.ftd

```diff
@@ -0,0 +1,128 @@
+
+-- ftd.type heading-hero-bold: $fpm.type.heading-hero.font
+desktop: $fpm.type.heading-hero.desktop
+mobile: $fpm.type.heading-hero.mobile
+xl: $fpm.type.heading-hero.xl
+weight: 700
+
+-- ft.page: `intro` and it's Usage
+
+`-- intro-two` can be used as part of blog posts.
+
+-- ft.code: Example code for `-- intro-two`
+lang: ftd
+
+\-- intro-two:
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minimium.
+
+
+
+
+-- intro-two:
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minimium.
+
+
+-- ftd.column intro-two:
+body body:
+ftd.color backgroundcolor: $fpm.color.main.background.base
+width: fill
+
+
+
+--- intro-desktop-italic:
```

Comment:
@priyanka9634 keep same code pattern in all files of this library!
refer `quote.ftd` and use same pattern for `desktop` and `mobile`